tests/unit/test_s3.py

```diff
@@ -0,0 +1,204 @@
+from unittest.mock import Mock, patch
+
+import pandas as pd
+
+from sdgym.s3 import is_s3_path, parse_s3_path, write_csv, write_file
+
+
+def test_is_s3_path_with_local_dir():
+    """Test the ``sdgym.s3.is_s3_path`` function with a local
+    directory. Expect that the return value is false.
+
+    Input:
+    - path
+
+    Output:
+    - boolean
+    """
+    # setup
+    path = 'path/to/local/dir'
+
+    # run
+    res = is_s3_path(path)
+
+    # asserts
+    assert not res
+
+
+def test_is_s3_path_with_s3_bucket():
+    """Test the ``sdgym.s3.is_s3_path`` function with an s3
+    directory. Expect that the return value is true.
+
+    Input:
+    - path
+
+    Output:
+    - boolean
+    """
+    # setup
+    path = 's3://my-bucket/my/path'
+
+    # run
+    res = is_s3_path(path)
+
+    # asserts
+    assert res
+
+
+def test_parse_s3_path_bucket_only():
+    """Test the ``sdgym.s3.parse_s3_path`` function with an s3 path
+    that contains only the bucket name. Expect that the aws bucket name
+    and key prefix are correctly parsed out.
+
+    Input:
+    - path
+
+    Output:
+    - bucket_name
+    - key_prefix
+    """
+    # setup
+    expected_bucket_name = 'my-bucket'
```

Comment:
I would define the `expected...` variables below, within the `assert` section of the test.